src/content/core/okteto-variables.mdx

```diff
@@ -7,36 +7,259 @@ id: okteto-variables
 
 import Image from "@theme/Image";
 
-Application configuration should be passed at deployment time, not hardcoded in your code.
+Configuration is a core concept of every application, we understand how important it is for developers and teams to parametrize their project deployments and Okteto manifests. At Okteto we built different ways for you and your teams to manage
+and share config values using Okteto Variables.
 
-Okteto variables help you save application configuration in Okteto and automatically inject it during deployment time.
+This page is a guide for those who are authoring Okteto manifests and configuring Okteto for their team and organization. If you are interested in runtime variables of dev environments, see [Development Containers](/docs/development/containers).
 
-There are a few different ways to leverage variables. In short:
+## What are Okteto Variables?
 
-- You can define variables manually in the manifest or in the Okteto UI
-- There are some default variables built in to the Okteto platform
-- All variables used in the Okteto manifest are automatically expanded
-- All variables are injected as runtime environment variables in the execution of your `deploy` and `destroy` scripts
+Okteto Variables are a way to parametrize your [Okteto manifest](/docs/reference/okteto-manifest) and the deploy, destroy and test strategy.
 
-We will cover each of these behaviors in more detail below.
+### Types of Variables
 
-## User-defined variables
+Each type is designed to solve a specific use-case.
 
-Okteto supports manual definition of variables, offering significant flexibility in configuration. These user-defined variables are separate from Okteto's default, built-in variables. We recommend creating variables in the following order of preference:
+- **Managed by Okteto**: variables injected in your dev environment automatically by Okteto
+- **The `--var` flag**: Used to define variables ad-hoc when using the Okteto CLI
+- **Catalog**: Used to define project-specific variables
+- **Local Environment Variables**: Used to define variables ad-hoc when using the Okteto CLI
+- **The `.env` file**: Used to define project or team-specific variables
+- **User Variables**: Used to define variables at the user level
+- **Admin Variables**: Used to define variables for all users in the organization
```

Comment:
Admin-defined variables? and maybe something like "variables defined by Okteto Admins than are injected in the dev environments of every user"